arches/app/signals.py

```diff
@@ -284,3 +284,49 @@ def ensure_single_default_searchview(sender, instance, **kwargs):
             raise ValidationError(
                 "Only one search logic component can be default at a time."
             )
+
+
+@receiver(post_save, sender=models.Card)
+@receiver(post_delete, sender=models.Card)
+@receiver(post_save, sender=models.CardModel)
+@receiver(post_delete, sender=models.CardModel)
```

Comment:
Ouch, nice catch that we have to double-up with the proxy models here. Looks like there's a draft PR against django to fix this.